### DISCUSSION TITLE
Релиз v1.4.0 | Beta

### DISCUSSION THREAD

**Discussion by DrIIvan** [Mister128/Noch.format#12] [Announcements]
Patch Note - v1.4.0 
Важные изменения
  - Добавлена возможность перелистывания заданий вниз, если их слишком много для отображения на экране.
  - При первом создании Word-файла и после смены акцентного цвета в настройках появляется предложение оставить обратную связь.

Изменения дизайна
  - При просмотре списка акцентных цветов каждое название будет отображаться с этим цветом.

Исправления и добавления в коде
  - Добавлены два параметра для preset.json ("used_once", "show_qiz").
  - Теперь для отображения списка условий заданий используется ListView, а не Column.
  - Перемещен "Базовый" цвет на позицию между "Чирок" и "Зеленый".
  - Добавлены переменные контроля действий пользователя в файл settings.py.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mister128/Noch.format/releases/tag/v1.4.0>Релиз v1.4.0</a>.</em>

**Comment by Mister128** [comment]
В следующий раз делай релизы в Announcements, а не Q&A =)
